cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -83,4 +89,46 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	setdefault.Register(plugins)
 	resize.Register(plugins)
 	pvcprotection.Register(plugins)
+
+	// Set recommended plugins order
+	recommendedOrder := []string{
+		deny.PluginName,
+		admit.PluginName,
+		autoprovision.PluginName,
+		lifecycle.PluginName,
+		initialization.PluginName,
+		mutatingwebhook.PluginName,
+		exists.PluginName,
+		scdeny.PluginName,
+		antiaffinity.PluginName,
+		initialresources.PluginName,
+		limitranger.PluginName,
+		serviceaccount.PluginName,
+		alwayspullimages.PluginName,
+		podpreset.PluginName,
+		podsecuritypolicy.PluginName,
+		podnodeselector.PluginName,
+		podpriority.PluginName,
+		podtolerationrestriction.PluginName,
+		defaulttolerationseconds.PluginName,
```

Comment:
this is out of order according to https://kubernetes.io/docs/admin/admission-controllers/#is-there-a-recommended-set-of-admission-controllers-to-use 

/cc @aveshagarwal @kevin-wangzefeng

Reply:
This are many mutating admission plugins, personally I think their absolute order does matter much if they are not conflicting.